developer-docs/digest-file.md

```diff
@@ -0,0 +1,68 @@
+# Digest File Reference
+
+This documentation serves as a technical reference for understanding the structure and purpose of the `digest` file.
+The purpose of this file is to enable secure and verifiable tracking of different versions of smart contracts and their associated methods.
+
+## File Structure
+
+The digest file consists of the following top-level fields:
+- `kompilation`: A string representing the SHA-256 hash of the compilation artifact.
+- `foundry`: A string representing the SHA-256 hash of the Foundry used to compile the contracts.
+- `methods`: An object containing key-value pairs, where each key represents a function name and an object with the following fields:
+- `method`: A string representing the SHA-256 hash of the method signature.
+- `contract`: A string representing the SHA-256 hash of the contract that contains the method.
+
+### Kompilation
+kompilation digest is computed here. It concatenates the hashes of `contracts.k`, `foundry.k` and any additional `.k` file passed to kontrol build using `--requires` (such as `lemmas.k`); then hashes the concatenated string once more.
```

Comment:
Hyperlinks missing?

Reply:
There weren't any in the original file here, but that's a good point — I'll add some to the files from our test suite.

Reply:
Sorry, I understand what you mean now — I added a missing link to the implementation and golden files from the test suite https://github.com/runtimeverification/gitbook-kontrol/pull/45/commits/2b0256eb06d9f997f9e42a9ac06c73bbfafaf441.